app/Spago/Dhall.hs

```diff
@@ -32,3 +35,10 @@ dhallToJSON inputPath outputPath = do
     $ Text.decodeUtf8
     $ ByteString.Lazy.toStrict
     $ JSON.encodePretty' config json
+
+
+addPackageToDhall :: T.FilePath -> T.Text -> IO ()
+addPackageToDhall file package = do
+    dhallText <- T.readTextFile file
+    expr <- Dhall.inputExpr dhallText
+    T.writeTextFile file dhallText
```

Comment:
Update: I looked into this, and what we'll need here is to be able to manipulate the Config in Dhall AST format, before actually trying to read it to Haskell (why? Because the import of the package-set will be resolved otherwise, and then when we write it back to file, we'll have aaaall the packages in the `spago.dhall`, which is not nice. So we have to keep the `Import` there).

So I'll implement a combinator `withConfigAST :: (Dhall.Expr -> Dhall.Expr) -> IO ()` (or something like that), to which you'll pass a function that manipulates the Config AST, and it will read it, apply the function, and write it back to file. This is necessary to implement #73 anyways.